Makefile

```diff
@@ -41,11 +41,17 @@ tls-tests:
 clean-redis:
 	docker rm --force ld-test-redis
 
+colon := :
 build-contract-tests:
 	@mkdir -p test-service/_checkouts
 	@rm -f $(CURDIR)/test-service/_checkouts/ldclient
 	@ln -sf $(CURDIR)/ $(CURDIR)/test-service/_checkouts/ldclient
-	@cd test-service && $(REBAR3) dialyzer
+	@if [ "$(OTP_VER)" = "26.x" ]; then\
```

Comment:
For now use a special profile for OTP 26 that turns off warnings for unknown functions.

If we can figure out some of the new dialyzer behavior, then maybe we can remove this.

Reply:
(Hopefully before OTP 27 arrives.)